package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sort-imports",
   "displayName": "sort-imports",
   "description": "Sort ES6 imports automatically.",
-  "version": "6.1.0",
+  "version": "6.1.1",
```

Comment:
don't bump the version yourself, the release process will do that

Reply:
Done, thanks.